dbt/tests/generic/README.md

```diff
@@ -0,0 +1,328 @@
+# Generic tests
+
+This directory stores the [custom generic
+tests](https://docs.getdbt.com/best-practices/writing-custom-generic-tests) that we use
+to define our test suite.
+
+## Available generic tests
+
+- [`test_accepted_range`](#test_accepted_range)
+- [`test_accepted_values`](#test_accepted_values)
+- [`test_column_is_subset_of_external_column`](#test_column_is_subset_of_external_column)
+- [`test_column_length`](#test_column_length)
+- [`test_columns_match`](#test_columns_match)
+- [`test_count_is_consistent`](#test_count_is_consistent)
+- [`test_expression_is_true`](#test_expression_is_true)
+- [`test_is_null`](#test_is_null)
+- [`test_no_extra_whitespace`](#test_no_extra_whitespace)
+- [`test_not_accepted_values`](#test_not_accepted_values)
+- [`test_not_null`](#test_not_null)
+- [`test_relationships`](#test_relationships)
+- [`test_res_class_matches_pardat`](#test_res_class_matches_pardat)
+- [`test_row_count`](#test_row_count)
+- [`test_row_values_match_after_join`](#test_row_values_match_after_join)
+- [`test_sequential_values`](#test_sequential_values)
+- [`test_unique_combination_of_columns`](#test_unique_combination_of_columns)
+- [`test_value_is_present`](#test_value_is_present)
```

Comment:
It would be nice if we could use some form of auto-documenter to generate this documentation based on the test docstrings, a la pkgdown. However, I think it's not worth spending too much time on that right now.

Reply:
**praise:** This is awesome and a lot of work! Very nice job. My only suggestion is to make it more prominent in the docs, perhaps by linking it under the Table of Contents and within the dbt catalog docs.

Reply:
I added a link to the README in [ea8fe36](https://github.com/ccao-data/data-architecture/commit/ea8fe36a25bbe25a777cbfdcaa9b34290d69583c)! I'm not totally sure where it would fit in the data catalog docs, do you have a suggestion?

Reply:
**demand (blocking):** This needs a better emoji before merge lol

Reply:
Agreed, what do you think about 🧪?

Reply:
I like it!

Reply:
Great, done in e3ba4366b1a4dc6418ab8451fbffb9d47082ef52!